src/magicbox/InputManager.ts

```diff
@@ -221,7 +221,6 @@ export class InputManager {
   }
 
   public focus() {
-    this.hasFocus = true;
```

Comment:
Ultimately, this is the one line of code that caused this issue.
I could simply remove it here, since it's set further in the code anyways.